tests/nnapi/nnapi_gtest.skip.aarch64-linux.acl_neon

```diff
@@ -96,6 +96,8 @@ GeneratedTests.logical_not_4D_nnfw
 GeneratedTests.logical_not_dynamic_nnfw
 GeneratedTests.logical_or_dynamic_nnfw
 GeneratedTests.logistic_dynamic_nnfw
+GeneratedTests.logistic_quant8_1
+GeneratedTests.logistic_quant8_2
```

Comment:
Please do not add these tests to `nnapi_gtest.skip.*.acl_cl` and `nnapi_gtest.skip.*.acl_neon`